src/components/articleSummary/ArticleSummary.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+  Card,
+  Row,
+  Col
+} from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import './articleSummary.scss';
+
+
+const ArticleSummary = (props) => {
+  const {
+    src, header, time, slug
+  } = props;
+  const date = new Date(time).getTime();
+  const now = new Date().getTime();
+  const hrs = Math.floor((now - date) / 1000 / 60 / 60);
+
+  return (
+    <Card className="card-outline">
+      <Card.Img style={{ height: '8rem' }} variant="top" src={src || 'https://res.cloudinary.com/djdsxql5q/image/upload/v1554806589/Authors%20Haven/culture.jpg'} />
```

Comment:
@Dubby20 you can determine the `src` and assign it to a variable, then use it here.